gateway/configuration/config.go

```diff
@@ -104,6 +104,9 @@ type GatewayConfig struct {
 	// Minimum ZookeeperTimeout is ('your Zookeeper server tickTime' * 2).
 	// Failover of targets will take (ZookeeperTimeout * 2) time.
 	ZookeeperTimeout time.Duration `json:"zookeeper_timeout"`
+	// EnableClustering will dictate if you want clustering turned on
```

Comment:
We're changing the default behavior. This might affect some existing clusters when upgrading, so we'll have to mention it in the commit message.